frontend/src/metabase/css/core/colors.module.css

```diff
@@ -2,7 +2,7 @@
  * NOTE: KEEP SYNCRONIZED WITH:
  * frontend/src/metabase/ui/utils/colors.ts
  * frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx
- * enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx
+ * frontend/src/metabase/styled-components/theme/css-variables.ts
```

Comment:
I want to remove the CSS variables from `GlobalStyles` and rely on `getMetabaseCssVariables`, but that's quite error-prone so let's do that in the next PR where I can focus on testing. If we do that, there will be a lot less duplication between GlobalStyles and SDK.